samples/cpp/tutorial_code/gapi/doc_snippets/dynamic_graph_snippets.cpp

```diff
@@ -1,6 +1,7 @@
 #include <opencv2/gapi.hpp>
 #include <opencv2/gapi/core.hpp>
 #include <opencv2/gapi/cpu/core.hpp>
+#include <opencv2/gapi/imgproc.hpp>
```

Comment:
Does the sample work correctly?

auto stream = cc.compileStreaming(cv::compile_args(cv::gapi::**core**::cpu::kernels()));

Reply:
Is it necessary to keep 
```
#include <opencv2/gapi/core.hpp>
#include <opencv2/gapi/cpu/core.hpp>
```
here?

Reply:
Applied, thanks